code/graphics/grstub.cpp

```diff
@@ -48,11 +48,6 @@ int gr_stub_save_screen()
 	return 1;
 }
 
-int gr_stub_tcache_set(int bitmap_id, int bitmap_type, float *u_scale, float *v_scale, int tex_unit = 0)
```

Comment:
`gr_opengl_tcache_set` is actually still being used by the OpenGL code internally so we need to keep that.